docs/architecture/porting-existing-aspnet-apps/incremental-migration-strategies.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Strategies for migrating incrementally
+description: What strategies can teams adopt that will allow them to migrate large applications from ASP.NET MVC to .NET Core in an incremental fashion?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Strategies for migrating incrementally
+
+The biggest challenge with migrating any large application is figuring out how to break the very large process up into smaller pieces. There are several strategies that can be applied for this purpose, including breaking up the app into horizontal layers such as UI, data access, business logic, or breaking up the app into separate, smaller apps. Another strategy that can help in this regard is to upgrade some or all of the application to different framework versions on the way to a recent .NET Core release.
+
+Consider the challenge of migrating a large ASP.NET 4.5 application. One approach would be to migrate directly from .NET 4.5 to .NET Core 3.1. However, this approach would need to account for every breaking change between these two frameworks and versions, which are substantial.
+
+One recent addition to the .NET ecosystem that helps with interoperability between different .NET frameworks is [.NET Standard](https://dotnet.microsoft.com/platform/dotnet-standard). The 2.0 .NET Standard release is notable because covers the vast majority of base class library functionality used by most .NET and .NET Core applications. Unfortunately, the earliest version of .NET with support for .NET Standard 2.0 is .NET 4.6.1.
+
+Thus, one approach to incrementally upgrade a .NET 4.5 system would be to first update its class libraries to .NET 4.6.1 and then to modify these libraries to be .NET Standard class libraries (using multi-targeting and conditional compilation, if required). Such libraries can be consumed by ASP.NET Core 2.1 apps, so the next step would be to migrate some or all of the web functionality of the app to ASP.NET Core 2.1 (as described in the [previous chapter](choose-net-core-version.md)).
```

Comment:
If libraries target NetStandard 2.0, the web app consuming them can move straight to ASP.NET Core 3.1. I don't think there's much value in using 2.1 as a stepping stone unless the customer has to stay on .NET Framework for a while (which isn't the case if their libraries target netstandard).